stock_location_package_restriction/models/stock_move.py

```diff
@@ -31,8 +31,9 @@ def _check_location_package_restriction(self):
         location_packages = {
             g["location_id"][0]: set(g["package_id"]) for g in quants_grouped
         }
+        lines_being_processed = self.move_line_ids.filtered(lambda line: line.qty_done)
```

Comment:
@TDu Is this working when not using the detailed operations and quantity_done directly on stock.move ?

Reply:
Thanks for your review and the answer is no, it is not working...

Reply:
It should also work as there are always move lines. See https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_move.py#L358

Reply:
My quick test was changing the unit test to update the quantity done  on the stock.move instead of the move line.
And it failed with 
`A package is mandatory on the location BWH/Stock/TestLocation2. You cannot move the product Wood without a package`
Which does not seem related to your question.